### PR TITLE
Improve stubs for traceback.FrameSummary

### PR DESCRIPTION
**Pull request by arturhoo**
Otherwise we get `'traceback.FrameSummary*' object is not iterable (Python 3)`

### CODE REVIEW

File: stdlib/2and3/traceback.pyi [hunk 1]
```diff
@@ -92,10 +92,19 @@ if sys.version_info >= (3, 5):
...
+        filename = ...  # type: str
+        lineno = ...  # type: int
+        name = ...  # type: str
+        locals = ...  # type: Any
         def __init__(self, filename: str, lineno: int, name: str,
                      lookup_line: bool = ...,
                      locals: Optional[Mapping[str, str]] = ...,
                      line: Optional[int] = ...) -> None: ...
+        def __eq__(self, other) -> bool: ...
+        def __getitem__(self, pos): ...
```
Comment (ambv) [inline stdlib/2and3/traceback.pyi]: Missing types.

File: stdlib/2and3/traceback.pyi [hunk 2]
```diff
@@ -92,10 +92,19 @@ if sys.version_info >= (3, 5):
...
+        def __iter__(self): ...
```
Comment (ambv) [inline stdlib/2and3/traceback.pyi]: Ditto.

File: stdlib/2and3/traceback.pyi [hunk 3]
```diff
@@ -92,10 +92,19 @@ if sys.version_info >= (3, 5):
...
+        @property
+        def line(self): ...
```
Comment (ambv) [inline stdlib/2and3/traceback.pyi]: Ditto.

**Comment by JelleZijlstra** [comment]
Thanks! It looks like after #1575 there's nothing left to do here.

**Comment by gvanrossum** [comment]
Sorry, I didn't know this was pending.

**Pull request closed without merging**